### PR TITLE
Fix: _globals schema nesting

### LINKED ISSUE #85: Schema issue - component settings not being displayed
Component settings are not being displayed on a v1 (rc2) AAT:

image

### LINKED ISSUE #91: Fix nesting on v1 schema _globals
### Subject of the issue/enhancement/features
The globals need to be nested inside a _components object.

### PR DESCRIPTION
**Pull request by joe-allen-89**
[//]: # (Please title your PR according to eslint commit conventions)
[//]: # (See https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-eslint#eslint-convention for details)

[//]: # (Link the PR to the original issue)
Fixes (#91 and #85)

[//]: # (Delete Fix, Update, New and/or Breaking sections as appropriate)
### Fix
* Nests globals schema within a _components object (fixes #91)
* Component schema $patch updated to $merge (fixes #85)

### CODE REVIEW

**Review by oliverfoster — APPROVED** [review]

**Review by cahirodoherty-learningpool — APPROVED** [review]

File: schema/component.schema.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "$anchor": "confidenceSlider-component",
   "$schema": "https://json-schema.org/draft/2020-12/schema",
   "type": "object",
-  "$patch": {
+  "$merge": {
```
Comment (guywillis) [inline schema/component.schema.json:5]: What's the requirement to change this from `$patch` to `$merge` or even, what does it do?

Comment (oliverfoster) [reply schema/component.schema.json:5]: They are meant to have different `with` formats when using them from their library [ajv-merge-patch](https://www.npmjs.com/package/ajv-merge-patch).
AATV1 Seems to treat them both as `$merge` style, as it implements it's own version of [`$merge and $patch`](https://github.com/adapt-security/adapt-authoring-jsonschema/blob/cd650d6bcff85fdb1b773f1b6fdee0b59333aff1/lib/JsonSchema.js#L170-L195
) and doesn't use the library. 

Ref: 
https://www.npmjs.com/package/ajv-merge-patch
https://ajv.js.org/guide/combining-schemas.html#merge-and-patch-keywords

Comment (guywillis) [reply schema/component.schema.json:5]: Thank you for the explanation

**Review by guywillis — APPROVED** [review]
👀

**Pull request merged**